plugins/alembic/CMakeLists.txt

```diff
@@ -1,4 +1,15 @@
-include(${CMAKE_SOURCE_DIR}/cmake/f3dPlugin.cmake)
+cmake_minimum_required(VERSION 3.19)
+
+project(f3d-plugin-alembic)
```

Comment:
Add a CI step building one internal plugin externally ?

Reply:
I'll do it later.